src/contracts/core/RewardsCoordinator.sol

```diff
@@ -271,6 +289,55 @@ contract RewardsCoordinator is
         _setGlobalOperatorCommission(_globalCommissionBips);
     }
 
+    /// @inheritdoc IRewardsCoordinator
+    function setOperatorAVSCommission(address operator, address avs, uint16 commission) external {
+        require(msg.sender == operator, "RewardsCoordinator.setOperatorAVSCommission: caller is not the operator");
+        require(
+            commission <= ONE_HUNDRED_IN_BIPS,
+            "RewardsCoordinator.setOperatorAVSCommission: commission must be <= 10000 bips"
+        );
+        uint32 activatedAt = SafeCast.toUint32(block.timestamp) + activationDelay;
```

Comment:
I've noticed we don't safe cast in other functions. So happy to remove if we feel strongly about it. `uint32` won't overflow anyway until ~2100 I think.

Reply:
Changed it back to casting using `uint32(*)` instead of Safecast: [4f65f54](https://github.com/Layr-Labs/eigenlayer-contracts/pull/837/commits/4f65f54414a387e5e00275c72a385f32c0cc023b)